datasketch/lsh.py

```diff
@@ -282,6 +301,36 @@ def _insert(
         for H, hashtable in zip(Hs, self.hashtables):
             hashtable.insert(H, key, buffer=buffer)
 
+    def __eq__(self, other:MinHashLSH) -> bool:
+        """
+        Returns:
+            bool: If the two MinHashLSH has equal num_perm then two are equivalent.
+        """
+        return (
```

Comment:
More checks need to be made. E.g., the b and r parameters.

Reply:
Added, b and r parameters. Please let me know if you think we need any other parameters too.